src/map/ai/controllers/mob_controller.h

```diff
@@ -25,6 +25,13 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 #include "controller.h"
 #include "entities/mobentity.h"
 
+enum FOLLOWTYPE
```

Comment:
Can we get this as an enum class, with the FollowType::None style naming?

Reply:
That's how I like to do it but I was trying to follow examples (pun intended)